torch/fx/graph_module.py

```diff
@@ -520,6 +526,9 @@ def __reduce_package__(self, exporter: PackageExporter):
         module_code = import_block + self.code
         exporter.save_source_string(generated_module_name, module_code)
 
+        if hasattr(self._graph, 'tracer'):
```

Comment:
I don't think this this should be a side effect of the `__reduce_package__` code. Let's just add it to the init of of `GraphModule` instead, so that all GraphModules retain a reference to their originating tracer.

Reply:
By raising an `Exception` in `__getstate__/__setstate__`?